data/tlg0081/tlg013/tlg0081.tlg013.1st1K-grc1.xml

```diff
@@ -0,0 +1,1192 @@
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Rebecca Frank</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Anita Galt</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Emily Graver</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Caroline Houser</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Serina Hu</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Peter Hulse</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Mert İnan</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Daria Kondakova</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Felipe Antonio Pinilla Torres</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Ruth Elizabeth Slater</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digital conversion and editing</resp>
+      </respStmt>
+      <respStmt>
+        <persName>Charlotte Houghton</persName>
+        <orgName>Center for Hellenic Studies</orgName>
+        <resp>Digial editor</resp>
+      </respStmt>
+    </titleStmt>
+    <publicationStmt>
+      <publisher>Open Greek and Latin</publisher>
+      <idno type="filename">tlg0081.tlg013.1st1K-grc1.xml</idno>
+      <pubPlace />
+      <availability>
+        <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available
+        under a Creative Commons Attribution-ShareAlike 4.0 International
+        License</licence>
+      </availability>
+      <date when="2021-05-25" />
+    </publicationStmt>
+    <sourceDesc>
+      <biblStruct>
+        <monogr>
+          <editor>Usener and Radermacher</editor>
+          <title xml:lang="lat">Dionysii Halicarnasei Opuscula 2.1</title>
+          <imprint>
+            <pubPlace>Leipzig</pubPlace>
+            <publisher>Teubner</publisher>
+            <date>1904</date>
+          </imprint>
+        </monogr>
```

Comment:
Please note the volume in the biblScope tag. Opuscula can be noted as a series.
Note that this is inconsistent with tlg0081.tlg014, so it might be good to revisit both files to come up with a uniform description.
I see that tlg0081.tlg014 is missing editor names.
We should have two lines:
```
<editor>first, last</editor>
<editor>first, last</editor>
```
This is how tlg0081.tlg014 reads and —while it needs the editor correction— use this as the model for this volume, too. (Note the series and volume info).
```
<biblStruct>
        <monogr>
          <title xml:lang="lat">De imitatione (fragmenta)</title>
          <author>Dionysius of Halicarnassus</author>
          <editor>Usener and Radermacher</editor>
          <imprint>
            <pubPlace>Leipzig</pubPlace>
            <publisher>Teubner</publisher>
            <date>1904</date>
          </imprint>
        </monogr>
        <series>
          <title xml:lang="lat">Opuscula</title>
          <biblScope unit="volume">2.1</biblScope>
        </series>
```